app/models/pipeline_run.rb

```diff
@@ -563,7 +563,12 @@ def db_load_amr_counts
                               allele: amr_result_fields[1],
                               coverage: amr_result_fields[2],
                               depth:  amr_result_fields[3],
-                              drug_family: amr_result_fields[12], }
+                              annotation_gene: amr_result_fields[11].split(";")[2],
+                              genbank_accession: amr_result_fields[11].split(";")[4],
+                              drug_family: amr_result_fields[12],
+                              total_reads: amr_result_fields[16],
+                              rpm: amr_result_fields[17],
+                              dpm: amr_result_fields[18].strip, } # Remove dangling newline character
```

Comment:
Not a big deal, but is this something we can easily get rid off in the pipeline output, so we don't need the special case here?

Reply:
No, since it's the last column in the csv file the entry will have a newline character.

I *could* rewrite the function to use Ruby's builtin CSV tools, which is honestly probably better...I think I'll do that.